src/components/OpenLayersMap.js

```diff
@@ -47,7 +47,7 @@ class OpenLayersMap extends React.Component{
   }
 
   renderTrails() {
-    const {trails, selectedTrail, hydrants} = this.props;
+    const {trails, selectedTrail, hydrants, mode} = this.props;
```

Comment:
It feels like we should be storing and adding features to the map in a Collection: https://openlayers.org/en/latest/apidoc/ol.Collection.html but I'm not sure.

Reply:
Yeah, something like that will be the right call. I think I'm going to try and see if I can update without storing any of those in state, just some sort of finding through the map object itself to avoid duplicating memory, but do it in a later PR for efficiency.